UnitsNet/QuantityTypeConverter.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+using System.ComponentModel;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+
+namespace UnitsNet
+{
+    /// <summary>
+    /// Is the base class for all attributes that are related to <see cref="UnitsNetTypeConverter{T}"/>
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
+    public abstract class UnitAttributeBase : Attribute
+    {
+        /// <summary>
+        /// This property hold the UnitType information
+        /// </summary>
+        public Enum UnitType { get; set; }
+
+        /// <summary>
+        /// Constructor of this class
```

Comment:
As boring as these are, I think the standard .NET naming is
`Initializes a new instance of the <see cref="foo"/> class.`

Same below.